Evaluation/plot_roc.py

```diff
@@ -41,7 +41,11 @@ def __init__(self, data, ref_roc=None, WPcurve=True):
 
         if ref_roc is None:
```

Comment:
I just realised that the colour overlap at 1.0 line might be happening because of this if statement on lines 42-43 (since it essentially always adds 1.0 line on the ratio, while it is useless). Would it resolve the problem if these lines are removed?

While disabling ratio plotting for WP-like curves can be a quick fix in the very short term, I think it should be considered in general. We also had quite an extensive discussion with @DennRoy in #79 on how to include this use case and I'd say it's worth keeping it.